src/libs/ajax/ajax-common.ts

```diff
@@ -67,21 +76,48 @@ export async function makeRequestRetry(request: Function, retryCount: number, ti
 
 const isUnauthorizedResponse = (error: unknown): boolean => error instanceof Response && error.status === 401;
 
+const createRequestWithStoredAuthToken = async (
+  wrappedFetch: FetchFn,
+  resource: RequestInfo | URL,
+  options?: RequestInit
+): Promise<Response> => {
+  const localToken = (await getAuthTokenFromLocalStorage())!;
+  const localHeaderJson = JSON.stringify(authOpts(localToken));
+  const memoryHeaderJson = JSON.stringify(authOpts());
+  if (localHeaderJson !== memoryHeaderJson) {
+    sendAuthTokenDesyncMetric();
+  }
```

Comment:
love this!

Reply:
Based on my multitab tests I have been manually doing, this gets tripped a fair bit especially when the computer falls asleep it seems.